sdk/appconfiguration/app-configuration/test/internal/helpers.spec.ts

```diff
@@ -130,6 +139,40 @@ describe("helper methods", () => {
     });
   });
 
+  describe("serializeAsConfigurationSettingParam", () => {
+    [[], `[]`, "Hello World"].forEach((value) => {
+      it(`serializer doesn't throw on ${value} as feature flag value`, () => {
+        const featureFlag: ConfigurationSetting<FeatureFlagValue> = {
+          contentType: featureFlagContentType,
+          key: "key",
+          isReadOnly: false,
+          value: { conditions: { clientFilters: [] }, enabled: true }
+        };
+        featureFlag.value = value as any;
+        assert.equal(
```

Comment:
Guessing this was also intended to be .deepEqual()

Reply:
Updated